src/poliastro/twobody/orbit.py

```diff
@@ -297,6 +297,13 @@ def circular(
             attractor, a, ecc, inc, raan, argp, arglat, epoch, plane
         )
 
+    @classmethod
+    def geostationary(cls):
```

Comment:
This method should allow the user to specify the attractor/body, and compute the geostationary orbit from there.

Reply:
Definitely. Will make required changes.

Reply:
@Juanlu001  I believe that for this we would need to add a new attribute(or inside kwargs dict) to Body class for storing period of one complete rotation on its axis. With this additional information we can find out geostationary orbit of any given Body(if exists).

Reply:
OK, as long as we don't use `hasattr` or similar inside the geostationary method to check for the property. For a simpler alternative, we can pass it as an extra parameter.

Reply:
Okay.. I proceed with an extra parameter for now.

Reply:
`Body` objects should not have methods that create `Orbit`s. `Body` objects should resemble a dumb `nametuple` as much as possible - the only exception now being the rotational elements, which should go away when we improve the reference frame objects.

And, as a more practical note, it would introduce a circular dependency because `orbit.py` imports `bodies`.

Reply:
Yes, that makes more sense. Thanks for the clarification.